### PR TITLE
CelebA dataset support added

### PR DESCRIPTION
**Pull request by anchitagarwal**

### CODE REVIEW

**Comment by ragavvenkatesan** [comment]
Can you check the last commit again ? There is something wrong. It fails the test. The commit before that is working properly.

File: yann/utils/dataset.py [hunk 1]
```diff
@@ -1480,7 +1480,7 @@ def _create_skdata_caltech256(self, verbose = 2): #pragma: no cover
         if verbose >=2:
                 print(".. Validation data")
         looper = n_valid_images / ( self.mini_batches_per_batch[2] * self.mini_batch_size )
-        for i in xrange(looper):        # for each batch_i file....
```
Comment (ragavvenkatesan) [inline yann/utils/dataset.py]: Why have you done these?

**Comment by anchitagarwal** [comment]
Nope. Just added my code. I double checked using diffmerge.
I pulled the dev branch. Is it possible I checked out the wrong branch?

**Comment by ragavvenkatesan** [comment]
Can you make this PR into V1 branch ?

**Pull request closed without merging**